quality/pre-push-recommended.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# We recommend you run this as a pre-push hook: to reduce
+# review turn-around time, we want all pushes to run tests
+# locally. Using this hook will guarantee your hook gets
+# updated as the repository changes.
+#
+# This hook tries to run as much as possible without taking
+# too long.
+#
+# You can use it by running this command from the project root:
+# `ln -s ../../quality/pre-push-recommended.sh .git/hooks/pre-push`
+
+# Descriptions for each gradle task below can be found in the
+# output of `./gradlew tasks`.
+./gradlew -q \
+        checkstyle \
+        ktlint \
+        pmd \
+        detektCheck \
+        assembleFocusX86DebugAndroidTest \
```

Comment:
So we build the UI tests and then only run the unit tests?

Reply:
Talked to @mcomella and they do it this way in ffftv so that they can see that the UI tests at least compile before someone pushes a PR